bin/src/hickory-dns.rs

```diff
@@ -66,13 +66,15 @@ use hickory_server::store::forwarder::ForwardAuthority;
 use hickory_server::store::recursor::RecursiveAuthority;
 #[cfg(feature = "sqlite")]
 use hickory_server::store::sqlite::{SqliteAuthority, SqliteConfig};
+#[cfg(feature = "blocklist")]
+use hickory_server::store::blocklist::BlockListAuthority;
```

Comment:
Nit, the naming here feels inconsistent? If `blocklist` is considered a single word, the type should be called `BlocklistAuthority` instead of `BlockListAuthority`. (If it's two words, the feature should be called `block-list` and the module `block_list`.)

Reply:
I changed this.